README.md

```diff
@@ -1,14 +1,54 @@
 # The A11y Machine
 
-**The A11y Machine** is an **automated accessibility testing tool** which
-**crawls** and **tests** all pages of any web application. It validates pages
-against the following specifications/laws:
+**The A11y Machine** (or `a11ym` for short, spelled “alym”) is an **automated
+accessibility testing tool** which **crawls** and **tests** all pages of any web
+application, to finally produce detailed and efficient reports. It validates
+pages against the following specifications/laws:
 
   * [W3C Web Content Accessibility Guidelines](http://www.w3.org/TR/WCAG20/)
     (WCAG) 2.0, including A, AA and AAA levels ([understanding levels of
     conformance](http://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-levels-head)),
   * U.S. [Section 508](http://www.section508.gov/) legislation.
 
+## Table of contents
+
+* [Why?](#why)
+* [Installation](#installation)
+* [Usage](#usage)
+  * [Possible output](#possible-output)
+  * [How does it work?](#how-does-it-work)
+  * [Write your custom sniffers](#write-your-custom-sniffers)
+* [Roadmap and board](#roadmap-and-board)
+* [Authors and license](#authors-and-license)
+
+## Why?
+
+If **privacy** matters for you, you're likely to install The A11y Machine over
+any SaaS services: It runs locally so you don't need to send your code
+somewhere, you can test all parts of your application including the ones which
+requires an authentification (like a checkout, a back-office etc.)…
+
+Here is the pros and cons of The A11y Machine vs. other third-party/SaaS
+services put in a table:
+
+Properties | The A11y Machine | SaaS services
+-----------|------------------|---------------
+Can run locally              | yes | no
+Can test each patch          | yes | no (except if commit)
+Reduce the test loop         | yes | no (the loop is longer)
+Can test private code        | yes | no (you must send your code)
+Can test auth-required parts | yes | no
+Can crawl all your pages     | yes | no (it depends of your money)
+
+Accessibility is not only a concern for disabled people: The biggest disabled
+peoples are bots, like [DuckDuckGo](https://duckduckgo.com),
```

Comment:
not sure "bots" are "people".

Maybe : "bots can also be considered as such" (which doesn't really remove the people comparison, but attenuate it a bit)